server/src/test/java/org/opensearch/discovery/PeerFinderTests.java

```diff
@@ -165,20 +165,20 @@ public String toString() {
     }
 
     class TestPeerFinder extends PeerFinder {
-        DiscoveryNode discoveredMasterNode;
-        OptionalLong discoveredMasterTerm = OptionalLong.empty();
+        DiscoveryNode discoveredClusterManagerNode;
+        OptionalLong discoveredClusterManagerTerm = OptionalLong.empty();
 
         TestPeerFinder(Settings settings, TransportService transportService, TransportAddressConnector transportAddressConnector) {
             super(settings, transportService, transportAddressConnector, PeerFinderTests.this::resolveConfiguredHosts);
         }
 
         @Override
-        protected void onActiveMasterFound(DiscoveryNode masterNode, long term) {
+        protected void onActiveMasterFound(DiscoveryNode clusterManagerNode, long term) {
```

Comment:
`onActiveMasterFound ` -> `onActiveClusterManager`

Reply:
Changed the variable name. 👍